submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+Meanwhile, the effects of temperature on the reactor itself were also softened: At first it was hinted that a nuclear reactor that heated up uncontrollably would explode, hence reactors would demand constant monitoring as in real life. This idea was softened before release though, such that a nuclear plants uses its internal emergency cooling systems to never let it itself heat up beyond 1000C, effectively allowing you to safely leave it alone. Well, mostly. Beware that it still *can* explode if is at over 900C and gets destroyed by a reckless player or biter. And if other reactors are close by, this may lead to [quite a light show](https://www.youtube.com/watch?v=l4ePIuhbiPg&ab_channel=Trupen). Despite the trivialized safety, we can see that Wube has provided a nuclear system which could still get pretty interesting and risky via the added complexity from the neighbor bonus. For those who would have preferred the high-risk and more realistic approach, there are multiple [mods]() that offer a look into the problems of meltdowns and radiation hazards.
+
+ Now that we' seen how nuclear power is not so dangerous in the vanilla game, let's look over the other common concerns about it.
+
+## **Common Concerns with Nuclear Power**
+
+* **Nuclear power is too expensive to set up until the late game** – In terms of research, you do not need more than chemical science packs to unlock nuclear power technology. The setting up requires time, yes. And a single nuclear reactor sure is expensive. However the material costs, in terms of iron and copper, for a 40MW starter level nuclear setup is only [a fraction]()<!--[See appendix]--> of the cost of setting up 40MW worth of solar panels and accumulators, and the cost per MW of adding any further reactors is substantially cheaper thanks to already-established infrastructure and the substantial gains from the neighbor bonus. 
+* **A nuclear reactor is too expensive to maintain without the Kovarex enrichment process** – Indeed, obtaining one unit of Uranium-235 requires centrifuging thousands of uranium ores, but on the other hand 1 piece of it is enough to make 10 uranium fuel cells, which last over 30 minutes in total. In a rather eye-opening [video](https://youtu.be/Qw_NzPuccxk?t=1610) Nilaus walks us through the mathematics and from there we see that you need only 8-9 miners and 1 centrifuge to maintain 1 nuclear reactor continuously without the Kovarex enrichment process, although it does not hurt to process a little more in case of bad luck during centrifuging.

```

Comment:
I like this point :) You could start the explanation though with saying that some players are intimidated by Kovarex, think it's required for nuclear, and then rather not try it. However, you actually don't need to run Kovarex at al to maintain a nuclear power plant...